django/db/migrations/writer.py

```diff
@@ -350,6 +350,8 @@ def serialize(cls, value):
         # process.
         if isinstance(value, Promise):
             value = force_text(value)
+        elif isinstance(value, LazyObject):
+            value = value.__reduce__()[1][0]
```

Comment:
I tried adding it at first but I used this approach for two reasons.

1. We want to avoid adding methods to `LazyObject` as it is really just a proxy.
2. [`deconstruct()`](https://docs.djangoproject.com/en/1.8/topics/migrations/#adding-a-deconstruct-method) can't simply return `self._wrapped` but has to figure the appropriate `(path, args, kwargs)` it must return. This is not an issue when a deconstructible object is wrapped but it is when it's not as all the special casing below demonstrate (e.g. a `re.compile` object).